### PR TITLE
Parse parameters and put them in tables

### PR DESCRIPTION
**Pull request by sornas**

### CODE REVIEW

File: doc-generator.py [hunk 1]
```diff
@@ -28,12 +28,25 @@ def format_doc(i, x):
...
+            if cmd == "code":
+                formatted = highlight(block).strip()
+                x = x[:block_start] + formatted + x[block_end+len(end_glyph):]
+            elif cmd == "params":
+                params = []
+                param_start = block.index("\n[")
+                while param_start != -1:
+                    param_end = block.index("]", param_start)
+                    param = block[param_start+len("\n["):param_end]
+                    param_start = block.find("\n[", param_end)
```
Comment (FredTheDino) [inline doc-generator.py]: This language is simple enough to be recognized by a regex. Maybe:
`r"\[(.*)\]\W*(.*)"`? (I didn't test it so make sure it works)

File: style.css [hunk 2]
```diff
@@ -14,16 +14,16 @@ body {
...
     border-left: 2px solid #EAE;
     padding-left: 1em;
     margin-top: 0.5em;
     margin-bottom: 0.5em;
 }
 
-.arg-name {
-    width: 5em;
-    display: inline-block;
+td {
```
Comment (FredTheDino) [inline style.css]: `.args td`

**Review by FredTheDino — APPROVED** [review]
Nice extension to the language, you can simplify some bits though! :D

**Comment by sornas** [comment]
The base is set to #8 so you can squash-merge if the regex looks good

**Pull request merged**